src/prefabs/questionnaire/checkboxWidget.ts

```diff
@@ -0,0 +1,229 @@
+              componentId: '#checkboxInput',
+              optionId: '#checkboxInputLabel',
+            },
+          },
+          configuration: {
+            showOnDrop: true,
+          },
+        }),
+        required: linked({
+          label: 'Required to answer',
+          value: {
+            ref: {
+              componentId: '#checkboxInput',
+              optionId: '#checkboxInputRequired',
+            },
+          },
+        }),
+        questionSpacing: linked({
+          label: 'Question spacing',
+          value: {
+            ref: {
+              componentId: '#questionBox',
+              optionId: '#questionBoxOuterSpacing',
+            },
+          },
+        }),
+        displayLogic: linked({
+          label: 'Display logic',
+          value: {
+            ref: {
+              componentId: '#questionBox',
+              optionId: '#questionBoxDisplayLogic',
+            },
+          },
+        }),
+      },
+    },
+    [
+      Box(
+        {
+          ref: {
+            id: '#questionBox',
+          },
+          options: {
+            ...boxOptions,
+            backgroundColor: color('Background color', {
+              value: ThemeColor.WHITE,
+            }),
+            borderColor: color('Border color', {
+              value: ThemeColor.MEDIUM,
+            }),
+            borderWidth: size('Border thickness', {
+              value: '1px',
+              configuration: {
+                as: 'UNIT',
+              },
+            }),
+            borderRadius: size('Border radius', {
+              value: '5px',
+              configuration: {
+                as: 'UNIT',
+              },
+            }),
+            outerSpacing: sizes('Outer space', {
+              value: ['0rem', '0rem', 'M', '0rem'],
+              ref: {
+                id: '#questionBoxOuterSpacing',
+              },
+            }),
+            displayLogic: displayLogic('Display logic', {
+              value: {},
+              ref: {
+                id: '#questionBoxDisplayLogic',
+              },
+            }),
+          },
+        },
+        [
+          component(
+            'Form',
+            {
+              ref: { id: '#checkboxQuestionForm' },
+              options: {
+                ...formOptions,
+                actionId: option('ACTION_JS', {
+                  label: 'Action',
+                  value: '',
+                  configuration: {
+                    createAction: {
+                      template: 'update',
+                      permissions: 'inherit',
+                    },
+                  },
+                }),
+              },
+            },
+            [
+              Text(
+                {
+                  ref: {
+                    id: '#questionText',
+                  },
+                  options: {
+                    ...textOptions,
+                    content: variable('Question text', {
+                      value: [''],
```

Comment:
Unfortunately this is not possible. I validated it on the text component by replacing the export with
```typescript
export default prefab('Text', attr, undefined, [
  Text(
    {
      options: {
        ...textOptions,
        type: font('Text style', { value: ['Body1'] }),
        useInnerHtml: toggle('Display Rich Text', { value: false }),
        content: variable('Content', {
          ...textOptions.content,
          value: ['hi mom'],
        }),
      },
    },
    [],
  ),
]);
```
If it would work, you would see the following key value pair in the executePrefab mutation 
```typescript
configuration: { as: 'MULTILINE', allowPropertyName: true },
```